### DISCUSSION TITLE
Where is the skeleton?

### DISCUSSION THREAD

**Discussion by hryang123** [talmolab/sleap#2079] [Help!]
Hello, I opened up sleap in hopes of labeling frames, but there skeleton option at the bottom seems to be missing. What can I do about this? 
![Screenshot 2024-12-30 120542](https://github.com/user-attachments/assets/2127790e-a7a3-4c6a-89b8-232f0bca790c)

**Comment by talmo** [comment]
Hi @hryang123,

You can get the Skeleton panel open again by selecting it from the bottom of the **View** menu in the menu bar at the top.

Cheers,

Talmo
